model/src/w3src4md.F90

```diff
@@ -294,17 +293,18 @@ SUBROUTINE W3SPR4 (A, CG, WN, EMEAN, FMEAN, FMEAN1, WNMEAN,     &
     DO IK=1, NK
       EB(IK)  = 0.
       EB2(IK) = 0.
+      SIGFAC=SIG(IK)**SSDSC(12) * DDEN(IK) / CG(IK)
```

Comment:
There's never a scenario where CG(IK) =0 is there?  (I'm assuming no, but if so we should have some sort of catch so we do not divide by zero).